src/dune_tui/dune_tui.ml

```diff
@@ -378,7 +378,15 @@ module Console_backend = struct
 end
 
 let backend =
-  let t = lazy (Dune_threaded_console.make (module Console_backend)) in
+  let t =
+    lazy
+      (Dune_threaded_console.make
+         ~frames_per_second:
+           (match Dune_config.Config.(get threaded_console_frames_per_second) with
+            | `Custom fps -> fps
+            | `Default -> 60)
```

Comment:
Why not respect the emacs config here as well?

Reply:
Initially because I was unsure if TUI would work inside emacs.

Reply:
Sure, but why create this inconsistency? I don't see what benefit it could have other than confusing people.

Reply:
Fixed.